docs/clients/fluentd/README.md

```diff
@@ -210,6 +210,21 @@ Specify a pair of client certificate and private key with `cert` and `key` if a
 </match>
 ```
 
+### server certificate verification
+A flag to enable a server certificate verification. By default the `verify_tls` is set to true.
```

Comment:
"to disable server verification"? :)

Reply:
This explanation should be fine. The reason why we introduce this flag is to specify server TLS verification mode. Not for disabling the verification. If we think, "to disable" is better, we need to change the option to `no_tls_verify` or something.

Reply:
I think your thinking around this patch came off the wrong foot. You seem to have thought that we somehow want to be able to _enable_ server certificate verification. It *is* enabled by default and we really look for a way to optionally disable it :).

Reply:
So, let's focus on the actual patch first and then adjust the docs, ok?